package.json

```diff
@@ -582,6 +582,7 @@
     "hasha": "^3.0.0",
     "js-yaml": "^3.12.1",
     "opn": "^5.4.0",
+    "path": "^0.12.7",
```

Comment:
path is in nodejs by default, don't use the npm module, especially when it's this out of date

Reply:
Thanks, @jrichter1 :)